frontend_app_admin_workspace_user/src/container/AdminWorkspaceUser.jsx

```diff
@@ -393,16 +382,36 @@ class AdminWorkspaceUser extends React.Component {
     GLOBAL_dispatchEvent({ type: CUSTOM_EVENT.SHOW_CREATE_WORKSPACE_POPUP, data: {} })
   }
 
-  handleChangeUsername = async (username) => {
+  handleChangeUsername = async (newUsername) => {
     const { props, state } = this
 
-    const fetchUsernameAvailability = await handleFetchResult(await getUsernameAvailability(state.config.apiUrl, removeAtInUsername(username)))
+    const username = removeAtInUsername(newUsername)
+    const fetchUsernameAvailability = await handleFetchResult(await getUsernameAvailability(state.config.apiUrl, username))
 
     switch (fetchUsernameAvailability.apiResponse.status) {
       case 200:
-        this.setState({ newUsernameAvailability: fetchUsernameAvailability.body.available })
+        this.setState({
+          isUsernameValid: fetchUsernameAvailability.body.available,
+          usernameInvalidMsg: props.t('This username is not available')
+        })
         break
-      default: this.sendGlobalFlashMsg(props.t('Error while checking username availability'))
+      default:
+        this.sendGlobalFlashMsg(props.t('Error while checking username availability'))
+        break
+    }
+
+    if (/\s/.test(username)) {
```

Comment:
same comment as Account and adminAccount

Reply:
Changed